hack/generator/pkg/config/type_transformer.go

```diff
@@ -17,65 +18,96 @@ import (
 type TransformTarget struct {
 	PackagePath string `yaml:",omitempty"`
 	Name        string `yaml:",omitempty"`
+	Map         *MapType
```

Comment:
I assume that `Map` is mutually exclusive with `Name` -- do we need to check that somewhere?

Reply:
Added check for this.